function heatUp(): void {
    throw new Error('no heat available');
}


try {
    heatUp();
} catch (err) {
    
}